ui/src/views/Tasks/ViewTask/task.graphql

```diff
@@ -68,4 +68,16 @@ query Task($taskId: ID!, $artifactsConnection: PageConnection, $taskActionsFilte
       name
     }
   }
+
+  dependents(taskId: $taskId, connection: $dependentsConnection, filter: $dependentsFilter) {
+    taskId
+    tasks {
+      status {
+        state
+      }
+      task {
+        task
+      }
```

Comment:
To fix the issue you are seeing, we'll want to specify the fields that we want the backend to return to us (e.g., task name, and state). Also, since this is a paginated endpoint, we'll want to receive pagination metadata in order to be able to fetch next/previous pages. Maybe something like the following will work:


```graphql
dependents(taskId: $taskId, connection: $dependentsConnection, filter: $dependentsFilter) {
  pageInfo {
    hasNextPage
    hasPreviousPage
    cursor
    previousCursor
    nextCursor
  }

  edges {
    node {
      taskId
      status {
        state
      }
      metadata {
        name
      }
    }
  }
}
```